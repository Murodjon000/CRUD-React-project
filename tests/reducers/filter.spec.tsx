import filterProject from "../../src/reducers/filter";

describe("filter reducer", () => {
  it("should return the initial state", () => {
    expect(
      filterProject(undefined, {
        type: "",
        payload: { value: "All" },
      })
    ).toMatchSnapshot();
  });

  it("should handle CHANGE_FILTER", () => {
    expect(
      filterProject("All", {
        type: "CHANGE_FILTER",
        payload: { value: "All" },
      })
    ).toMatchSnapshot();
  });
});
