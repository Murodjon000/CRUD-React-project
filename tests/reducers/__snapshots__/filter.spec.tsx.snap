// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`filter reducer should handle CHANGE_FILTER 1`] = `"All"`;

exports[`filter reducer should return the initial state 1`] = `"All"`;
